benchmarks/configs/unsupervised_inference_experiments.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+import copy
+import os
+from dataclasses import asdict
+
+from benchmarks.configs.defaults import default_evidence_lm_config, pretrained_dir
+from benchmarks.configs.names import UnsupervisedInferenceExperiments
+from tbp.monty.frameworks.config_utils.config_args import (
+    DetailedEvidenceLMLoggingConfig,
+    MontyArgs,
+    MotorSystemConfigCurInformedSurfaceGoalStateDriven,
+    PatchAndViewSOTAMontyConfig,
+    SurfaceAndViewSOTAMontyConfig,
+    get_cube_face_and_corner_views_rotations,
+)
+from tbp.monty.frameworks.config_utils.make_dataset_configs import (
+    EnvironmentDataloaderPerObjectArgs,
+    EvalExperimentArgs,
+    PredefinedObjectInitializer,
+    get_env_dataloader_per_object_by_idx,
+)
+from tbp.monty.frameworks.environments import embodied_data as ED
+from tbp.monty.frameworks.experiments import MontyObjectRecognitionExperiment
+from tbp.monty.frameworks.loggers.monty_handlers import (
+    BasicCSVStatsHandler,
+    DetailedJSONHandler,
+    ReproduceEpisodeHandler,
+)
+from tbp.monty.frameworks.models.evidence_unsupervised_inference_matching import (
+    MontyForUnsupervisedEvidenceGraphMatching,
+)
+from tbp.monty.simulators.habitat.configs import (
+    PatchViewFinderMountHabitatDatasetArgs,
+    SurfaceViewFinderMountHabitatDatasetArgs,
+)
+
+"""
+The configs provided in this file are used for unsupervised inference experiments.
+
+These experiments are simpler versions of the benchmark experiments designed for fast
+prototyping and testing. More specifically, they are configured for less number of
+rotations, less number of objects, and no noise. These configs call
+`MontyForUnsupervisedEvidenceGraphMatching`, which removes explicit reset logic.
+"""
+
+# === Configs parameters section === #
+
+# Changes the number of rotations per object
+num_rotations = 1
```

Comment:
These look like constants to me, are they ever going to be changed by the code below? If not, then stylistically they should be all caps to call attention to that fact.